pychunkedgraph/app/segmentation/common.py

```diff
@@ -382,7 +385,7 @@ def handle_merge(table_id):
     current_app.logger.debug(("lvl2_nodes:", ret.new_lvl2_ids))
 
     if len(ret.new_lvl2_ids) > 0:
-        publish_edit(table_id, ret.new_lvl2_ids, is_priority=is_priority)
+        publish_edit(table_id, user_id, ret, is_priority=is_priority)
```

Comment:
isn't the user id already part of `ret`?

Reply:
no user id comes from request, operation result only has `Result = namedtuple("Result", ["operation_id", "new_root_ids", "new_lvl2_ids"])`